docs/install.rst

```diff
@@ -281,33 +338,63 @@ you want to access, you will have to get the certificates from the corresponding
 information is available in their user guide.
 
 
+.. _troubleshooting:
+
 Troubleshooting
 =======================
 
 **Missing virtualenv**
 
-This should return the version of the RADICAL-Pilot installation, e.g., `0.X.Y`.
+This should return the version of the RADICAL-Pilot installation, e.g., `1.0.0`.
 
 If virtualenv **is not** installed on your system, you can try the following.
 
 .. code-block:: bash
 
-        wget --no-check-certificate https://pypi.python.org/packages/source/v/virtualenv/virtualenv-1.9.tar.gz
-        tar xzf virtualenv-1.9.tar.gz
+        wget --no-check-certificate http://pypi.python.org/packages/source/v/virtualenv/virtualenv-16.7.9.tar.gz
+        tar xzf virtualenv-16.7.9.tar.gz
+
+        python virtualenv-16.7.9/virtualenv.py $HOME/ve-entk -p python3.7
+        source $HOME/ve-entk/bin/activate
+
+**Python 2 legacy installation**
+
+As of January 1, 2020, Python 2 support is terminated by the Python Software
+Foundation but the previous release of EnTK i.e. ``0.72.1`` allows to use Python 2.
+PyPI installation with virtualenv is:
+
+.. code-block:: bash
+
+        virtualenv $HOME/ve-entk-py2 -p python2.7
+        source $HOME/ve-entk-py2/bin/activate
+        pip install radical.entk==0.72.1
+
+```radical-stack``` confirms the versions of the radical cybertools:
+
+.. code-block:: bash
+
+        $ radical-stack
+
+          python               : 2.7.17
+          pythonpath           :
+          virtualenv           : /home/username/ve-entk-py2
 
-        python virtualenv-1.9/virtualenv.py $HOME/myenv
-        source $HOME/myenv/bin/activate
+          radical.entk         : 0.72.1
+          radical.pilot        : 0.73.1
+          radical.saga         : 0.72.1
+          radical.utils        : 0.72.0
 
-**TypeError: 'NoneType' object is not callable**
+.. comments
```

Comment:
I would eliminate this comment as we do not support python 2 anymore and above, we have mentioned only python 2.7 as the legacy version supported.